### PR TITLE
Close #1666: Add wappalyzer as easy

### LINKED ISSUE #1666: Add Pizza Hut
I have been trying to delete a pizzahut.com account for sometime. Going to the "contact us" didn't really do much. I had the the idea that maybe I could delete my account through the Pizza Hut app. That was one of the ways that closing an account is possible. I had to fill out the information (name, address, phone number, etc.) associated with the account and closed the account. 

- `name`: pizzahut.com
- `url`: Download the Pizza Hut app. Sign-in with account information. Click on the "account" tab, tap on profile. The close my account section is in this section. Tap on the close my account. Fill in your information (name, address, phone, email,). Click submit.
- `difficulty`: Hard
- `notes`: Download Pizza Hut app. Or call customer service at  1-800-948-8488 
- `domains`: This is used by our [Browser Extensions](https://github.com/jdm-contrib).

### PR DESCRIPTION
**Pull request by mbtrs**
Issue #1166

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -13200,7 +13210,7 @@
             "wordpress.com"
         ]
     },
-
+    
     {
```
Comment (tupaschoal) [inline _data/sites.json:13214]:
```suggestion

     {
```

File: _data/sites.json [hunk 2]
```diff
@@ -12817,6 +12817,16 @@
...
+    {
+        "name": "wappalyzer",
+        "url": "https://www.wappalyzer.com/account/",
+        "difficulty": "easy",
+        "notes": "Click on your e-mail on the top right in the navigator. Find and click on Account. Scroll down to find the delete account button in red. Confirm deletion. Enter password to complete deletion.",
+        "domains": [
+            "wappalyzer.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:12829]:
```suggestion
    },

```

**Review by tupaschoal — COMMENTED** [review]
Seems fair now, just a few whitespaces to be removed

File: _data/sites.json [hunk 3]
```diff
@@ -12817,6 +12817,15 @@
...
 
+    {
+        "name": "wappalyzer",
+        "url": "https://www.wappalyzer.com/account/",
+        "difficulty": "easy",
+        "notes": "Click on your e-mail on the top right in the navigator. Find and click on Account. Scroll down to find the delete account button in red. Confirm deletion. Enter password to complete deletion.",
+        "domains": [
+            "wappalyzer.com"
+        ]
+    },
```
Comment (tupaschoal) [inline _data/sites.json:12828]:
```suggestion
    },

```

Comment (tupaschoal) [reply _data/sites.json:12828]: Sadly I reverted the whitespaces, but also the line break and can't fix it on mobile. Can you add just a line break here without whitespaces, please, like the suggestion @mbtrs ?

**Review by mbtrs — COMMENTED** [review]
I'm not sure why the test is failing. I did another commit earlier to adjust some of the whitespace that may have occured but it isn't nearly as much as it is reporting. I've tried a test on my own through an IDE and found that there was no trailing whitespace issues. I've taken the now newly cleaned up json file on my IDE and copy and pasted it into sites.json file on my fork and there showed no changes to the file.

**Review by tupaschoal — APPROVED** [review]
All well now :)

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**